.golangci.yml

```diff
@@ -1,15 +1,30 @@
+version: "2"
 run:
   # disallows implicit automatic updating of go.mod and fails when any changes to go.mod are needed
   modules-download-mode: readonly
+
 linters:
   enable:
-    - errcheck        # Checks for unchecked errors
-    - gosimple        # Finds code simplifications
-    - govet           # Vet checks for various errors
-    - ineffassign     # Finds useless assignments
-    - staticcheck     # Comprehensive static analysis
-    - unused          # Finds unused variables, functions, etc.
-    - gosec           # Security checks for Go code
-    - goimports       # Checks import statements are formatted
-    - gofmt           # Checks whether code was gofmt-ed
+    - asciicheck
+    - bidichk
+    - durationcheck
+    - errname
+    - errorlint
+    - govet
```

Comment:
It is not a problem to have it explicit, but it is enabled by default: https://golangci-lint.run/usage/linters/#enabled-by-default

Reply:
Maybe we could also include `gosec`?

Reply:
Thanks for the feedback @marcusburghardt. I added `gosec` as well.